apps/prairielearn/src/schemas/infoAssessment.ts

```diff
@@ -1,13 +1,7 @@
-import { type ZodSchema, z } from 'zod';
+import { z } from 'zod';
```

Comment:
We could consider doing the schema migrations separately, using the `/v4` import path for an incremental migration.